integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/partition/AlterTableSplitCarbonPartitionCommand.scala

```diff
@@ -109,7 +109,7 @@ case class AlterTableSplitCarbonPartitionCommand(
     CarbonUtil.writeThriftTableToSchemaFile(schemaFilePath, thriftTable)
     // update the schema modified time
     carbonMetaStore.updateAndTouchSchemasUpdatedTime(storePath)
-    sparkSession.catalog.refreshTable(tableName)
+    sparkSession.catalog.refreshTable(s"$dbName.$tableName")
```

Comment:
Better to call SessionCatalog.refreshTable rather than Catalog refreshTable as it will honour the DBName.
sparkSession.sessionState.catalog.refreshTable() and pass TableIdentifier.

Reply:
@sounakr Thank you for your review.  I will fix this.